core/src/main/scala/kafka/coordinator/group/CoordinatorPartitionWriter.scala

```diff
@@ -201,18 +203,57 @@ class CoordinatorPartitionWriter[T](
     ))
   }
 
+  /**
+   * Verify the transaction.
+   *
+   * @param tp              The partition to write records to.
+   * @param transactionalId The transactional id.
+   * @param producerId      The producer id.
+   * @param producerEpoch   The producer epoch.
+   * @return A future containing the {@link VerificationGuard} or an exception if the
```

Comment:
nit: did we want to update this comment to be consistent with the PartitionWriter one?

Reply:
sorry, missed it. i think that i will just replace those docs with links to the interface doc in order to avoid this in the future.

Reply:
done.